start-frontend.sh

```
@@ -0,0 +1,34 @@
+
+# Kyros Dashboard - Frontend Quick Start Script
+echo "🚀 Starting Kyros Dashboard Frontend..."
+
+# Check if we're in the right directory
+if [ ! -f "ui/package.json" ]; then
+    echo "❌ Error: Please run this script from the project root directory"
+    exit 1
+fi
+
+# Navigate to UI directory
+cd ui
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)